frontend/src/routes/Infrastructure/BareMetalAssets/BareMetalAssetsPage.css

```diff
@@ -0,0 +1,11 @@
+.titleBMA_deprecated {
```

Comment:
this whole file is not needed if you use `label` instead of `description` for ACMPageHeader